bitnami/ghost/templates/deployment.yaml

```diff
@@ -1,197 +1,189 @@
-          value: {{ .Values.mariadb.db.name | quote }}
-        {{- else }}
-          value: {{ .Values.externalDatabase.database | quote }}
-        {{- end }}
-        - name: GHOST_DATABASE_USER
-        {{- if .Values.mariadb.enabled }}
-          value: {{ .Values.mariadb.db.user | quote }}
-        {{- else }}
-          value: {{ .Values.externalDatabase.user | quote }}
-        {{- end }}
-        - name: GHOST_DATABASE_PASSWORD
-        {{- if .Values.mariadb.enabled }}
-          valueFrom:
-            secretKeyRef:
-              name: {{ template "ghost.mariadb.fullname" . }}
-              key: mariadb-password
-        {{- else }}
-          value: {{ .Values.externalDatabase.password | quote }}
-        {{- end }}
-        - name: GHOST_HOST
-          value: {{ include "ghost.host" . | quote }}
-        - name: GHOST_PROTOCOL
-          value: {{  .Values.ghostProtocol | quote }}
-        - name: GHOST_PORT_NUMBER
-        {{- if .Values.ghostPort }}
-          value: {{ .Values.ghostPort | quote }}
-        {{- else }}
-          value: {{ .Values.service.port | quote }}
-        {{- end }}
-        - name: GHOST_USERNAME
-          value: {{ .Values.ghostUsername | quote }}
-        - name: GHOST_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: {{ template "ghost.fullname" . }}
-              key: ghost-password
-        - name: GHOST_EMAIL
-          value: {{ .Values.ghostEmail | quote }}
-        - name: BLOG_TITLE
-          value: {{ .Values.ghostBlogTitle | quote }}
-        {{- if .Values.smtpHost }}
-        - name: SMTP_HOST
-          value: {{ .Values.smtpHost | quote }}
-        {{- end }}
-        {{- if .Values.smtpPort }}
-        - name: SMTP_PORT
-          value: {{ .Values.smtpPort | quote }}
-        {{- end }}
-        {{- if .Values.smtpUser }}
-        - name: SMTP_USER
-          value: {{ .Values.smtpUser | quote }}
-        {{- end }}
-        {{- if .Values.smtpPassword }}
-        - name: SMTP_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: {{ template "ghost.fullname" . }}
-              key: smtp-password
-        {{- end }}
-        {{- if .Values.smtpFromAddress }}
-        - name: SMTP_FROM_ADDRESS
-          value: {{ .Values.smtpFromAddress | quote }}
-        {{- end }}
-        {{- if .Values.smtpService }}
-        - name: SMTP_SERVICE
-          value: {{ .Values.smtpService | quote }}
-        {{- end }}
-        ports:
-        - name: http
-          containerPort: 2368
-        {{- if .Values.livenessProbe.enabled }}
-        livenessProbe:
-          httpGet:
-            path: {{ .Values.ghostPath }}
-            port: http
-            httpHeaders:
-            - name: Host
-              value: {{ include "ghost.host" . | quote }}
-            {{- if eq .Values.ghostProtocol "https" }}
-            - name: X-Forwarded-Proto
-              value: https
+        - name: {{ include "ghost.fullname" . }}
```

Comment:
I'm not in favour of having a container with the name of the release. It doesn't make much sense to me and it is more problematic in terms of debugging. In my opinion, the container name should be `ghost`

Reply:
```suggestion
        - name: ghost
```

Reply:
You are right, it was a mistake. ;)